packages/opentelemetry-types/src/metrics/Metric.ts

```diff
@@ -41,8 +44,26 @@ export interface MetricOptions {
   /** The resource the Metric is associated with. */
   resource?: Resource;
 
-  /** The name of the component that reports the Metric. */
-  component?: string;
+  /** Indicates the metric is a verbose metric that is disabled by default */
+  disabled?: boolean;
+
+  /**
+   * Bidirectional allows counter metrics to accept negative values.
+   * Otherwise, as false, the counter is unidirectional and rejects negative values.
```

Comment:
Could you explicitly call out the default value (which I assume is false, since omitting it will trigger `undefined`, a falsey value)? Same for other options below to say the default if not specified.